nextflow_schema.json

```diff
@@ -829,6 +829,29 @@
                 }
             }
         },
+        "host_removal": {
+            "title": "Host Removal",
+            "type": "object",
+            "description": "",
+            "default": "",
+            "properties": {
+                "run_host_removal": {
+                    "type": "boolean",
+                    "description": "Turn on per-lane creation pre-Adapter Removal FASTQ files without reads that mapped to reference (e.g. for public upload of privacy sensitive non-host data)",
+                    "help_text": "Create pre-Adapter Removal FASTQ files without reads that mapped to reference (e.g. for public upload of privacy-sensitive non-host data)",
+                    "fa_icon": "fas fa-power-off"
+                },
+                "host_removal_mode": {
+                    "type": "string",
+                    "default": "remove",
+                    "description": "Host-mapped read removal mode. Remove mapped reads completely from FASTQ (remove) or just mask the sequence of mapped reads with N (replace).",
```

Comment:
```suggestion
                    "description": "Host-mapped read removal mode. Remove mapped reads completely from FASTQ (remove) or just mask the host sequence of mapped reads with N (replace).",
```

Reply:
Could add a extended `help_text` saying, can be useful to set when you want to make sure you have the same number of reads as the input data to your host-removed publically uploaded data, but still removing any host information.